src/dotnet/commands/dotnet-compile-fsc/Program.cs

```diff
@@ -170,7 +170,7 @@ public static int Run(string[] args)
                 //HACK we need default.win32manifest for exe
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 {
-                    var win32manifestPath = Path.Combine(AppContext.BaseDirectory, "default.win32manifest");
+                    var win32manifestPath = Path.Combine(AppContext.BaseDirectory, "runtimes", "any", "native", "default.win32manifest");
```

Comment:
AppContext.Basedirectory is redirected in tools scenario, I used location of the assembly to get around this.